### DISCUSSION TITLE
blog/how-to-integrate-spotify-api-with-next-js

### DISCUSSION THREAD

**Discussion by giscus** [ibizabroker/tarungowda.com#9] [Announcements]
# blog/how-to-integrate-spotify-api-with-next-js

Learn how to integrate Spotify API with Nextjs. In this blog I show three examples for Spotify's Web API, get your last song played, your top 5 artists and tracks for the last 4 weeks.

https://www.tarungowda.com/blog/how-to-integrate-spotify-api-with-next-js

**Comment by ibizabroker** [comment]
Recently tried to generate the refresh token and I got the error `INVALID_CLIENT: Insecure redirect URI`, I noticed spotify has updated their docs. 

change the redirect URI from `http://localhost:3000/` -> `http://127.0.0.1:3000/` for it to work.
